src/components/atoms/Image/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './style.css';
+
+const Image = ({ src, alt, className } ) => {
+    return(
+        <img src={src} alt={alt} className={className} />
```

Comment:
what the purpose of this component?

Reply:
I wanted to make all the elements to components. Maybe it is needless.

Reply:
if component brings no value it should not be impolemented